_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+Not only in our smartphones and cars we can find chips but also in areas such as
+
+- Personal computing (PC, laptops, smartphones)
+- Entertainment (TVs, radios, gaming consoles, ...)
+- Health care
+- Industry/manufacturing
+- Planes
+- Rockets
+- Mars rovers (yes, we even have chips on _other planets_!)
+- ...
+
+We can probably continue that list endlessly. But you get the point: It's hard
+to imagine modern society without microchips!
+
+Ok, now you should have a broad understanding of what a microchip is and why
+they are so important nowadays. So your next question probably is: How are they
+made? This is where Digital Design comes into play. Digital design is a branch
+of electrical and computer engineering that focuses on the design of, well, you
+guessed it, microchips. Therefore, we also call it "Chip Design". But before we
+dive into the design of chips, we first need to understand what a chip is
+actually made of.
+
+## Transistors
+
+<figure style="width:40%" class="align-right">
+  <img src="/assets/images/dd-00001-mosfet.png" alt="">
+  <figcaption class="text-center">
+  A transistor. (by <a href="https://commons.wikimedia.org/wiki/User:Brews_ohare">Brews ohare</a>,
+  <a href="https://commons.wikimedia.org/wiki/File:MOSFET_Structure.png">MOSFET
+  Structure</a>,
+  <a href="https://creativecommons.org/licenses/by-sa/3.0/legalcode" rel="license">CC
+  BY-SA 3.0</a>)
+  </figcaption>
+</figure>
+
+Let's start from the foundations. A chip is nothing but a tiny electrical
+circuit. The fundamental building block that only made microchips possible is
+the **transistor**. Maybe you have heard about them before (processor marketing
+likes to include statements like "_Our new processor has XY billion
+transistors!_"). A transistor is nothing but a _switch_. However, unlike a
+_mechanical_ switch, a transistor's switching behavior can be controlled by some
+other _current_ (or voltage). This allows the transistor to be much faster and
+smaller.
+
+The first transistor was invented way back in 1947 (!) and could be held in one
+hand. Since then the sizes of transistors have been shrinking - and they still
+do today. Modern chips contain _billions_ of transistors. When you hear terms
+like "5nm CPU", that means nothing but the size of one transistor on that chip
+(actually that is not entirely true, but that's another story). So, transistors
+nowadays are only in the range of one _billionth_ of a meter in size! You might
+have heard about "Moore's Law" before. The "law" was named after Intel
+co-founder Gordon Moore. In 1965, he predicted that the number of transistors in
+chips would double about every two years. And in fact, this _exponential_ trend
+kept continuing for several decades, thanks to the advancements in transistor
+technology.
+
+<figure style="width:60%" class="align-center">
+  <img src="/assets/images/dd-00001-transistors-per-microprocessor.svg" alt="">
+  <!-- <figcaption class="text-center"></figcaption> -->
+</figure>
```

Comment:
Do you have an alternative image. On my computer it looks good but on my laptop it does not look nice. The lines and font are not thick enough.